hooks/useGroupFacetFilters.js

```diff
@@ -0,0 +1,18 @@
+import { gql, useQuery } from '@apollo/client';
+
+export const GET_LINEUPS = gql`
```

Comment:
Nitpick, leftover copy/paste it looks like?

```suggestion
export const GET_GROUP_FACET_FILTERS = gql`
```